resources/test/rest_schema_status.json

```diff
@@ -5,36 +5,44 @@
   "type": "object",
   "required": [
     "api_version",
+    "available_block_range",
```

Comment:
Please also check and update tests in `casper-test`. These two are probably affected:
* `rpc_info_get_status.rs`
* `rpc_list_rpcs.rs`